doc/whats_new/v0.22.rst

```diff
@@ -493,6 +493,10 @@ Changelog
   ``multioutput`` parameter.
   :pr:`14732` by :user:`Agamemnon Krasoulis <agamemnonc>`.
 
+- |Fix| The scorers: 'roc_auc_ovr', 'roc_auc_ovo', 'roc_auc_ovr_weighted', 
```

Comment:
Since these were never released, it doesn't really make sense to advertise as a separate change log entry does it?

Reply:
I had not realized that. Indeed we should remove that changelog entry.

Reply:
I had not realized that. Indeed we should remove that changelog entry.